bin/aapp_dr_runner.py

```diff
@@ -706,19 +706,36 @@ def generate_process_config(msg, config):
 
     config['start_time'] = msg.data['start_time']
 
+    # Save collection_area_id if given
+    try:
+        config['collection_area_id'] = msg.data['collection_area_id']
```

Comment:
This can be simplified, like this: 
```suggestion
        config['collection_area_id'] = msg.data.get('collection_area_id', None)
```

Reply:
The two lines be low should also be removed.. (I am not sure how to do this in here)

Reply:
yes, you can do it locally and the push it.